app/controllers/reminders_controller.rb

```diff
@@ -1,26 +1,87 @@
 class RemindersController < BasePublicController
+  helper_method :current_reminder
+
   def new
-    @reminder = Reminder.new
-    render "personal-details".underscore
+    render first_template_in_sequence
   end
 
   def create
-    @reminder = Reminder.new(reminder_params)
-
-    if @reminder.save(context: :"personal-details")
-      redirect_to reminder_path
+    current_reminder.attributes = reminder_params
+    if current_reminder.save(context: current_slug.to_sym)
+      session[:reminder_id] = current_reminder.to_param
+      redirect_to reminder_path(current_policy_routing_name, next_slug)
     else
-      render "personal-details".underscore
+      render first_template_in_sequence
     end
   end
 
   def show
-    render "eligible-later-completion".underscore
+    render current_template
+  end
+
+  def update
+    current_reminder.attributes = reminder_params
+    one_time_password
+    if current_reminder.save(context: current_slug.to_sym)
+      redirect_to reminder_path(current_policy_routing_name, next_slug)
+    else
+      show
+    end
   end
 
   private
 
+  def first_template_in_sequence
+    Reminder::SLUGS.first.underscore
+  end
+
+  def current_template
+    current_slug.underscore
+  end
+
+  def next_template
+    next_slug.underscore
+  end
+
+  def next_slug
+    Reminder::SLUGS[current_slug_index + 1]
+  end
+
+  def current_slug
+    Reminder::SLUGS[current_slug_index]
+  end
+
+  def current_slug_index
+    Reminder::SLUGS.index(params[:slug]) || 0
+  end
+
+  def current_policy_routing_name
+    "early-career-payments"
```

Comment:
@d-a-v-e Just a question, if we want to extend reminders at a point in the future I assume we could alter this to be policy determination similar to how its done for each Policy?